### PR TITLE
[Closed] Index of Alphabet In C implementation, Readme Changes

### LINKED ISSUE #1035: Write a Julia programme to find volume of cylinder
### Description

Write a programme to find volume of cylinder

Volume of cylinder = π * (radius)^2 * height

```
radius : 2
height : 3

volume : 37.69911
```

### LINKED ISSUE #1137: Write a C programme to find index of an alphabet
### Description

Write a programme to find index of an alphabet

> **Note** index starts from 1

```
Input  : C
Output : 3
```

### PR DESCRIPTION
**Pull request by SpirosArk**
### Why:

Closes #1137 
Closes #1035 

### Check off the following:

- [x ] I have reviewed my changes in staging (look for "Automatically generated comment" and click **Modified** to view your latest changes).
- [x ] For content changes, I have completed the [self-review checklist](https://github.com/codinasion/codinasion/blob/master/contributing/self-review.md).

### CODE REVIEW

File: programme/index-of-alphabet/index-of-alphabet.c [hunk 1]
```diff
@@ -0,0 +1,24 @@
+//Program that accepts a char as an input and returns it's position in alphabet
```
Comment (KikiManjaro) [inline programme/index-of-alphabet/index-of-alphabet.c:1]:
```suggestion
//Programme that accepts a char as an input and returns it's position in alphabet
```

File: programme/volume-of-cylinder/volume-of-cylinder.jl [hunk 2]
```diff
@@ -0,0 +1,8 @@
+#Program to calculate volumeOfCylinder
```
Comment (KikiManjaro) [inline programme/volume-of-cylinder/volume-of-cylinder.jl:1]:
```suggestion
#Programme to calculate volumeOfCylinder
```

**Review by KikiManjaro — CHANGES_REQUESTED** [review]
Minor fixes related to #1295

**Comment by SpirosArk** [comment]
I accepted suggestions please rereview

File: programme/index-of-alphabet/index-of-alphabet.c [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    if (input >= 97){
+        
+        position = input - 'a' + 1;
+        
+    }
+    else{
+    
+        position = input - 'A' + 1;
+    
+    }
```
Comment (KikiManjaro) [inline programme/index-of-alphabet/index-of-alphabet.c:20]:
```suggestion
    if (input >= 97){
        position = input - 'a' + 1;
    }
    else{
        position = input - 'A' + 1;
    }
```

**Comment by KikiManjaro** [comment]
You've made a mistake and modified volume in this MR, this one should only refer to index-of-alphabet

**Comment by SpirosArk** [comment]
I accepted all suggested commits cause i saw them from cellphone

**Review by KikiManjaro — APPROVED** [review]

**Comment by harshraj8843** [comment]
Tracked in #1294

**Pull request closed without merging**